frontend/src/employee-frontend/components/reports/VoucherServiceProviderUnit.tsx

```diff
@@ -407,14 +407,10 @@ function VoucherServiceProviderUnit() {
                       <Td>{row.childGroupName}</Td>
                       <Td>
                         <Tooltip
-                          position="top"
-                          tooltip={
-                            <div>
-                              {`${row.realizedPeriod.start.format()} - ${row.realizedPeriod.end.format()}`}
-                            </div>
-                          }
+                          tooltip={<div>{row.numberOfDays}</div>}
+                          position="right"
                         >
-                          {row.numberOfDays}
+                          {row.realizedPeriod.format()}
```

Comment:
If I remember right, here the period is always within one month so if we need to save space we could also format `dd.mm.yyyy - DD.mm.yyyy` as `dd.-DD.mm.yyyy`

Reply:
Good idea 👍